### ISSUE TITLE
Unclear what "this" refers to in "For regression, this is a scalar..."

### ISSUE THREAD

**Issue by gitclem** [probml/pml-book#39]
p559 (589 of PDF) 2020-01-03 Draft

3rd paragraph:

> (For regression, **this** is a scalar; for classification, **it** can be the logits or
> class probabilities.)

It could be clearer as to what "this" and "it" is.  (I presume theta?)

**Comment by murphyk** [comment]
fixed
